docs/ecommerce-applications/shopware-6/how-to-configure-remote-storage-for-shopware-6-x.md

```diff
@@ -0,0 +1,107 @@
+
+```yaml
+shopware:
+    filesystem:
+      public:
+        type: 'amazon-s3'
+        config:
+            bucket: 'main'
+            region: 'EU'
+            endpoint: 'management url'
+            use_path_style_endpoint: true
+            credentials:
+              key: 'access key'
+              secret: 'secret key'
+
+```
+
+You can get the required management URL, access key, and secret key by running `hypernode-object-storage info` with the `--with-credentials` flag:
+
+```console
+app@testapp ~ # hypernode-object-storage info --with-credentials
++--------------------------------------+----------------+---------+-------------+-------------------------------------+---------------+---------------+
+|                 UUID                 |      Name      |   Plan  |  Hypernodes |           Management URL            |   Access Key  |   Secret Key  |
++--------------------------------------+----------------+---------+-------------+-------------------------------------+---------------+---------------+
+| 12345678-9012-3456-b7e3-19ab43df4a23 | testappbucket1 | OS200GB |   testapp   |  https://example.ams.objectstore.eu |   abcd1234    |   abcd1234    |
++--------------------------------------+----------------+---------+-------------+-------------------------------------+---------------+---------------+
+```
+
+**AWS S3**
+
+If you're using Amazon's S3 bucket, your configuration file should include the following:
+
+```yaml
+shopware:
+    filesystem:
+      public:
+        type: 'amazon-s3'
+        config:
+            bucket: 'your bucket name'
+            region: 'your aws region'
+            credentials:
+              key: 'your aws access key'
+              secret: 'your aws secret key'
+
+```
+
+## Syncing the files
+
+You can use the following to sync the files between your local and remote storage:
+
+```bash
+hypernode-object-storage objects sync public/media/ s3://bucket_name/media/
+```
+
+In the case of Hypernode Object Storage, the bucket name will always be `main`.
+
+The `hypernode-object-storage objects sync` command runs the sync process in the background
+and provides the Process ID (PID). You can monitor the sync progress using:
+
+```bash
+hypernode-object-storage objects show PID
+```
+
+Alternatively, you can use the AWS CLI directly:
+
+```bash
+aws s3 sync public/media/ s3://bucket_name/media/
+```
+
+## Serving assets from your S3 bucket
+
+To serve media assets directly from your S3 bucket, you need to adjust your nginx configuration.
+Fortunately, `hypernode-manage-vhosts` [simplifies this process for you](../../hypernode-platform/nginx/hypernode-managed-vhosts.md#object-storage-and-hypernode-managed-vhosts).
+
+### Configuring Amazon S3 bucket policies
+
+If you’re using Amazon S3, ensure that your S3 bucket has ACLs enabled.
+
+## Shopware remote storage documentation
+
+- [Filesystem](https://developer.shopware.com/docs/guides/hosting/infrastructure/filesystem.html)
```

Comment:
Since it's just one link, I think it's better to move this one to the `Configuring the application` section